Lazy_Lists.md

```diff
@@ -0,0 +1,41 @@
+# Lazy Lists
+
+Lazy lists are one of the fundamental aspects in Functional Programming because it allows us to define and create infinite lists.  In Coconut, Lazy lists, represented by “banana brackets” `(| |)`, are defined to be lazily evaluated iterator literals. This means that values are not stored in the list, but iterables are. Lazy evaluation is similar to the `::` operator, in that expressions are not evaluated until that particular element is needed. 
+
+In a simple Python program, we might write a recursive program for Fibonacci by doing 
```

Comment:
Maybe we should intro the Fibonacci problem first?

Reply:
Yeah—as a user, my reaction to this would be "what's Fibonacci?"